index.html

```diff
@@ -24542,13 +24542,39 @@ <h2 id="acknowledgments">
 
 
 <section class="appendix" id="revision_log">
-  <h2>Revision Log</h2>
+  <h2>
+  <span its-locale-filter-list="en" lang="en">Revision Log</span>
+  <span its-locale-filter-list="ja" lang="ja">変更記録</span>
+  </h2>
   <p its-locale-filter-list="en" lang="en">The following changes were made since the previous publication.</p>
+  <p its-locale-filter-list="ja" lang="ja">前回の公開時から以下の変更が行われています．</p>
   <ul>
-    <li id="id692">TBD</li>
+    <li>
+      <p its-locale-filter-list="en" lang="en">Merged and simplified <a href="#id237">handling of overhanging ruby text on punctuation marks</a> (former 3.3.8 f,g,h)</p>
+      <p its-locale-filter-list="ja" lang="ja"><a href="#id237">約物にかかるルビ文字の定義</a>を集約し簡単化しました (以前の 3.3.8 f,g,h)</p>
+      [<a href="https://github.com/w3c/jlreq/issues/15">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/44">PR</a>]</li>
+    <li>
+      <p its-locale-filter-list="en" lang="en">English wording fixes</p>
+      <p its-locale-filter-list="ja" lang="ja">英語表現の修正</p>
+      [<a href="https://github.com/w3c/jlreq/issues/10">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/66">PR</a>;
+      <a href="https://github.com/w3c/jlreq/issues/67">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/82">PR</a>;
+      <a href="https://github.com/w3c/jlreq/issues/85">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/91">PR</a>]</li>
+    <li>
+      <p its-locale-filter-list="en" lang="en">Merging English and Japanese version</p>
```

Comment:
I'm inclined to use the past tense. Also 'version' is a little vague.  Perhaps change to this?

> Merged English and Japanese documents into a single document, with switches to allow readers to view the text in either language, or both.

Reply:
Note, btw, that you used past tense in the bullet above, too.

Reply:
modified as suggested, and checked tense (and last period).